src/zoomifytilesource.js

```diff
@@ -0,0 +1,97 @@
+(function( $ ){
+
+  /**
+   * @class ZoomifyTileSource
+   * @classdesc A tilesource implementation for the zoomify format.
+   *
+   * @memberof OpenSeadragon
+   * @extends OpenSeadragon.TileSource
+   * @param {Number} width - the pixel width of the image.
+   * @param {Number} height
+   * @param {Number} tileSize
+   * @param {String} tilesUrl
+   */
+  $.ZoomifyTileSource = function( options ) {
```

Comment:
We're in the process of changing some aspects of our code style (see #456, mentioned in https://github.com/openseadragon/openseadragon/blob/master/CONTRIBUTING.md). One of those things is removing the spaces inside parentheses. Since this is a new file it would be great if it started out with the new style.